subprojects/osgi-test/src/testBundle/java/org/mockito/osgitest/testbundle/MockClassInOtherBundleTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.osgitest.testbundle;
+
+import org.mockito.osgitest.otherbundle.Methods;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MockClassInOtherBundleTest implements Runnable {
+
+    @Override
+    public void run() {
+        Methods methods = mock(Methods.class);
+        when(methods.intReturningMethod()).thenReturn(42);
+        if (methods.intReturningMethod() != 42) {
```

Comment:
Nit: use `assertThat` from AssertJ or `assertThrows` from JUnit 5 to verify the exception. That makes the test clearer and provides better error messages when the test fails.

Reply:
(In this case, an `assertEquals(methods.intReturningMethod(), 42)` would be better.

Reply:
Using `assertEquals(expected, actual)` now that it's available.